### DISCUSSION TITLE
Calico network policies being skipped in backups

### DISCUSSION THREAD

**Discussion by colinliddlemyob** [vmware-tanzu/velero#3270] [Community support Q&A] [answered]
**What steps did you take and what happened:**
Backing up and restoring a cluster with calico network policies

**What did you expect to happen:**
Both k8s network policies and calico network policies should be backed up and restored

**The output of the following commands will help us better understand what's going on**:

Creating a backup and just specifying the calico network policies will create a backup (but just with those resources)
```velero create backup --include-resources networkpolicies.crd.projectcalico.org ...```

Creating a backup without specifying resources (eg all resources) will skip over the calico network policies because it thinks it has already backed them up, but was actually just the k8s native network policies
```velero create backup ...```
``` time="2021-01-06T23:32:50Z" level=info msg="Skipping resource because it cohabitates and we've already processed it" backup=kube-system/mf-backup-9 cohabitatingResource1=networkpolicies.extensions cohabitatingResource2=networkpolicies.   networking.k8s.io group=crd.projectcalico.org/v1 logSource="pkg/backup/resource_backupper.go:148" resource=networkpolicies```

**Anything else you would like to add:**

If I try ```--include-resources *,networkpolicies.crd.projectcalico.org``` I get the error ```Validation errors:  Invalid included/excluded resource lists: includes list must either contain '*' only, or a non-empty list of items```

**Environment:**

- Velero version (use `velero version`): 1.3.1 (have tried 1.5.2 as well with same issues)
- Velero features (use `velero client config get features`): None
- Kubernetes version (use `kubectl version`): 1.17.12
- Kubernetes installer & version: Self managed
- Cloud provider or hardware configuration: AWS EC2
- OS (e.g. from `/etc/os-release`): Amazon linux 2

**Vote on this issue!**

This is an invitation to the Velero community to vote on issues, you can see the project's [top voted issues listed here](https://github.com/vmware-tanzu/velero/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc).  
Use the "reaction smiley face" up to the right of this comment to vote.

- :+1: for "I would like to see this bug fixed as soon as possible"
- :-1: for "There are more important bugs to focus on right now"

**Comment by carlisia** [comment, marked as answer]
This discussion has been moved to an issue: https://github.com/vmware-tanzu/velero/issues/3346.
